packages/zoe/src/contracts/newSwap/multipoolAutoswap.js

```diff
@@ -159,6 +160,15 @@ const start = zcf => {
     getPool,
   );
 
+  const { makeCollectFeesInvitation } = makeMakeCollectFeesInvitation(
```

Comment:
This seems a lot of ceremony for what could imply be an embedded `zcf.makeInvitation(collectFees, 'collect Fees');`. BUT perhaps that's a pattern we've established and so the ceremony is preferred?

Reply:
`collectFees` wants `feeSeat` and the brand to be in scope. `collectFees` could be directly in multipoolAutoswap to achieve that, but it seems better as a separate thing.